python/paddle/tensor/random.py

```diff
@@ -20,3 +20,87 @@
 #            'randperm',
 #            'rand',
 #            'randint']
+
+from __future__ import print_function
+
+import numpy as np
+import warnings
+import six
+import os
+import inspect
+from ..fluid.layer_helper import LayerHelper
+from ..fluid.initializer import Normal, Constant, NumpyArrayInitializer
+from ..fluid.framework import Variable, OpProtoHolder, in_dygraph_mode, dygraph_only, _dygraph_tracer, default_main_program, device_guard
+from ..fluid import dygraph_utils
+from ..fluid.param_attr import ParamAttr
+from ..fluid import unique_name
+from ..fluid import core
+from ..fluid.data_feeder import convert_dtype, check_variable_and_dtype, check_type, check_dtype
+from ..fluid.layers.nn import gaussian_random
+
+__all__ = ['randn']
+
+
+def randn(shape,
+          out=None,
+          dtype=None,
+          device=None,
+          stop_gradient=True,
+          name=None):
+    """
+    This function returns a tensor filled with random numbers from a normal distribution with mean 0 and variance 1 (also called the standard normal
+    distribution).
+
+    Args:
+        shape(list|tuple): Shape of the generated random tensor.
+        out(Variable, optional): Optional output which can be any created Variable that meets the requirements to store the result of operation. If
+            the out is `None`, a new Variable wiil be create to store the result. Default is None.
+        dtype(np.dtype|core.VarDesc.VarType|str, optional): Data type of the output tensor, which can be float16, float32, float64, int32, int64, bool.
+            if dtype is `None`, the data type of output tensor is `float32`. Default is None.
```

Comment:
Would you mind setting default dtype to 'float32'?

Reply:
The definition of the API is given in the table, although I personally think float32 will be more suitable.

![image](https://user-images.githubusercontent.com/52460041/77635863-2e3cd800-6f8e-11ea-97e9-6a40e0adfe89.png)

Reply:
Both 'float32' and None are fine to me. 'float32' is slightly better, but since this is our requirement, we can keep it as None